src/Item_Disk.php

```diff
@@ -713,8 +713,19 @@ public static function getSpecificValueToDisplay($field, $values, array $options
 
     public function getNonLoggedFields(): array
     {
-        return [
+        // we don't want to log at all changes of available space for a drive
+        // as it's likely to change every time
+        $exclude = [
             'freesize',
         ];
+
+        // logging total size of zfs mount points make no sense as it's equal to the used space of the point + available space for the pool
+        // it's likely to have this key changing on each automatic inventory
+        // so we don't want to pollute logs with these frequent changes
+        if (in_array(($this->input['filesystem'] ?? ""), ['zfs', 'fuse.zfs'])) {
```

Comment:
Maybe, in edge cases, `filesystem` will not be passed in input, and, in this case, we should use the current value.
```suggestion
        if (in_array(($this->fields['input'] ?? $this->fields['filesystem'] ?? ""), ['zfs', 'fuse.zfs'])) {
```

Reply:
I'm note sure about this, when we arrive at this code, if ->fields exists, it will contains data corresponding to db.
the table doesn't contains a filesystem field but a filesystems_id.

Reply:
Indeed.

Can you add a comment to explain that `$this->input['filesystem']` will be present only during inventory ?